tests/Frameworks/WordPress/Version_6_1/index.php

```diff
@@ -11,6 +11,7 @@
  *
  * @var bool
  */
+require __DIR__.'/../../../Appsec/Mock.php';
```

Comment:
Would you mind to move this above the docblock which talks about `WP_USE_THEMES` plz? Docblocks should be in the line above of what they talk about. Also, add a comment to why we add the mock. On other files where it's included the comment is `Appsec mock. This wont be needed on customer apps since this functions will be exposed by appsec`. You can use the same

Reply:
Please change this in all files where you added it 🙏